spec/features/ingredients/edit_ingredient_page_spec.rb

```diff
@@ -0,0 +1,27 @@
+feature 'user visits edit ingredients page' do
+  before do
+    FactoryBot.create(:ingredient, :egg, id: 1)
+    visit('/ingredients/1/edit')
```

Comment:
Can we start from `ingredients#index` and navigate to edit by clicking the button?